dbt_copilot_helper/commands/copilot.py

```diff
@@ -218,8 +219,8 @@ def make_addons():
 
 
 @copilot.command()
-@click.argument("app", type=str)
-@click.argument("env", type=str)
+@click.argument("app", type=str, required=True)
+@click.argument("env", type=str, required=True)
```

Comment:
Do we need a test for this since they are required?

Reply:
There are annoyingly no tests for this command at present. It's worth addressing, but not as part of this ticket I think.

Reply:
Yup, sounds like a separate ticket.